src/emulator/auth/cloudFunctions.ts

```diff
@@ -54,9 +61,16 @@ export class AuthCloudFunction {
   private createEventRequestBody(
     action: AuthCloudFunctionAction,
     userInfoPayload: UserInfoPayload
-  ): { eventType: string; data: UserInfoPayload } {
+  ): CreateEvent {
     return {
+      eventId: uuid.v4(),
       eventType: `providers/firebase.auth/eventTypes/user.${action}`,
+      resource: {
+        name: `projects/${this.projectId}`,
```

Comment:
Shall this include the user id? I'm not sure what the production behavior actually is though

Reply:
I was just following the docs here, but I will deploy a function quickly to confirm.

Reply:
Updated the description with a production payload.